io/pci/EEH.py

```diff
@@ -134,6 +158,22 @@ def test_eeh_basic_pe(self):
                                 break
                             else:
                                 self.log.info("PE recovered successfully")
+                    time.sleep(10)
+                    if (self.pci_class_name == 'scsi_host' and
```

Comment:
so no validation for other than scsi_host classes ?

Reply:
Will be addressing it in PR "https://github.com/avocado-framework-tests/avocado-misc-tests/pull/2590"